core/creators/themedui.js

```diff
@@ -239,6 +239,9 @@ CKEDITOR.replaceClass = 'ckeditor';
 
 			// Delay to avoid race conditions (setMode inside setMode).
 			setTimeout( function() {
+				if ( editor.status === 'destroyed' || editor.isDetached() ) {
```

Comment:
I wonder about introducing `editor.isDestroyed()`. Basically it would be just a nicer way to write `editor.status === 'destroyed'`.

Also I find

```javascript
if ( editor.isDestroyed() || editor.isDetached() )
```

more elegant than the current version.

Yes, I know, I have obsession about readability and elegance of code 😄